spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/annotation/ContinueSpan.java

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
I think that the users of Spring Data that want to have some stuff traced will be able to extend the interfaces and add the Sleuth repositories. Why would someone want to use these annotations without Sleuth?